### PR TITLE
Bletalien testcases

### PR DESCRIPTION
**Pull request by bletalien**
Trying to add a second pull request but since the first one hasn't been approved, Github isn't letting me? Or for some other reason? Either way, I did 1.1-1-fail-1 and 1.1-2-fail-1.

### CODE REVIEW

**Comment by bletalien** [comment]
For both 1.1-1 and 1.1-2, I think we need cases in which the code fails the first part of the condition (can access via keyboard, no timing required) but passes the second condition (alternative means on the page) and so it's a pass. I imagine some methodologies will miss that second part.

File: testcases.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Until we modify the primary index file, this serves as a placeholder with links
 
 Test Case documents are organized by Baseline Test (each serving as a Test Scenario or general "functional requirement" for testing), which inculde multiple Test Cases. Will need to develop a Test Scenario template to summarize Test Scenario purpose and subordinate Test Cases
 
+* `1.` Keyboard Accessible
+  * [Keyboard Access - Functionality not keyboard accessible (TC01.1-1-fail-1)](testcases/TC01.1.1-fail-1.html)
```
Comment (kengdoj) [inline testcases.md:13]: filename link should have a hyphen instead of another dot: TC01.1-1-fail-1.html

File: testcases/TC01.1-1-fail-1.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# The layout must be 'testcase'; DO NOT Change
+layout: testcase
+
+# Brief, descriptive title for the test case
+title: Keyboard Access - functionality not keyboard accessible
+
+# The Test Case ID should follow the pattern: 
+# TC[Baseline Test Procedure #]-[Test Instruction #]-
+# [Expected Result (pass/fail/dna)]-[example #], e.g., TC05.1-1-fail-1
+tcid: TC1.1-1-fail-1
```
Comment (kengdoj) [inline testcases/TC01.1-1-fail-1.md:11]: missing a zero: TC01.1-1-fail-1

File: testcases/TC01.1-2-fail-1.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    **Baseline Test ID:** 1.1-KeyboardAccess
+    
+    **Test Instruction:** 2
+
+# Expected result that the ICT Baseline would predict
+# [Pass | Fail | DNA]
+result: Fail
+
+# Brief description of the rationale for the expected result
+result-descr: > Users must press and hold a key for three seconds to use a function.
```
Comment (kengdoj) [inline testcases/TC01.1-2-fail-1.md:33]: I had YAML errors when building this page. 

- removing the `>` character fixed one of the errors

@bletalien - did you start with a blank template or use an existing testcase file?

**Review by kengdoj — CHANGES_REQUESTED** [review]

**Comment by kengdoj** [comment]
Great start @bletalien!!

File: testcases/TC01.1-1-fail-1.md (hunk 2 above)
Comment (drewnielson) [reply testcases/TC01.1-1-fail-1.md:11]: fixed

File: testcases.md (hunk 1 above)
Comment (drewnielson) [reply testcases.md:13]: fixed

File: testcases/TC01.1-2-fail-1.md (hunk 3 above)
Comment (drewnielson) [reply testcases/TC01.1-2-fail-1.md]: fixed

**Review by drewnielson — APPROVED** [review]
Approved with edits

**Pull request merged**